aspnet-core/src/AbpCompanyName.AbpProjectName.Application/AppConsts.cs

```diff
@@ -6,5 +6,7 @@ public class AppConsts
         /// Default pass phrase for SimpleStringCipher decrypt/encrypt operations
         /// </summary>
         public const string DefaultPassPhrase = "gsKxGZ012HLL3MI5";
+
+        public const string ClientRootAddress = "http://localhost:4200";
```

Comment:
Better way to get client url, you should get it from appsettings.json by using IConfiguration.

Reply:
@alirizaadiyahsi can you help me out on this one? Should I inject one of these?

![image](https://user-images.githubusercontent.com/5401232/55186254-dc902080-5174-11e9-9211-f3034b927110.png)